### ISSUE TITLE
Quilter fails to launch if file in sidebar no longer exists

### ISSUE THREAD

**Issue by bebeki** [lainsce/quilter#236]
I have used Quilter on and off in the past, but I've kept the app uninstalled because it consistently overwrites my files. Today I installed the latest version (1.8.0) to see if the app was now safe to use. 

When I reinstalled the app, the sidebar was prepopulated with files I had previously opened. I proceeded to close these files.

One of the files in the sidebar no longer existed at the location that Quilter remembered. I had moved it to a new location during the time when Quilter was uninstalled. When I attempted to close that file within Quilter's sidebar, the app crashed. If I tried to launch Quilter again, the app failed to launch. 

Here is what appeared when I launched Quilter in the terminal (which is the only way I figured out what the problem was):

```
com.github.lainsce.quilter %U

(com.github.lainsce.quilter:4801): Gtk-CRITICAL **: 17:33:24.505: gtk_header_bar_pack: assertion 'gtk_widget_get_parent (widget) == NULL' failed

(com.github.lainsce.quilter:4801): Gtk-CRITICAL **: 17:33:24.506: gtk_header_bar_pack: assertion 'gtk_widget_get_parent (widget) == NULL' failed

(com.github.lainsce.quilter:4801): Gtk-CRITICAL **: 17:33:24.507: gtk_header_bar_pack: assertion 'gtk_widget_get_parent (widget) == NULL' failed

** (com.github.lainsce.quilter:4801): CRITICAL **: 17:33:24.564: gtk_spell_checker_set_language: assertion 'GTK_SPELL_IS_CHECKER (spell)' failed

** (com.github.lainsce.quilter:4801): CRITICAL **: 17:33:24.564: gtk_spell_checker_attach: assertion 'GTK_SPELL_IS_CHECKER (spell)' failed

** (com.github.lainsce.quilter:4801): CRITICAL **: 17:33:24.564: quilter_main_window_dynamic_margins: assertion 'self != NULL' failed

(com.github.lainsce.quilter:4801): Gtk-CRITICAL **: 17:33:24.598: gtk_header_bar_pack: assertion 'gtk_widget_get_parent (widget) == NULL' failed

** (com.github.lainsce.quilter:4801): CRITICAL **: 17:33:24.599: quilter_main_window_dynamic_margins: assertion 'self != NULL' failed

(com.github.lainsce.quilter:4801): Gtk-CRITICAL **: 17:33:24.617: gtk_header_bar_pack: assertion 'gtk_widget_get_parent (widget) == NULL' failed

** (com.github.lainsce.quilter:4801): CRITICAL **: 17:33:24.619: quilter_main_window_dynamic_margins: assertion 'self != NULL' failed

(com.github.lainsce.quilter:4801): Gtk-CRITICAL **: 17:33:24.645: gtk_header_bar_pack: assertion 'gtk_widget_get_parent (widget) == NULL' failed

** (com.github.lainsce.quilter:4801): CRITICAL **: 17:33:24.646: quilter_main_window_dynamic_margins: assertion 'self != NULL' failed

(com.github.lainsce.quilter:4801): Gtk-CRITICAL **: 17:33:24.665: gtk_header_bar_pack: assertion 'gtk_widget_get_parent (widget) == NULL' failed

** (com.github.lainsce.quilter:4801): CRITICAL **: 17:33:24.666: quilter_main_window_dynamic_margins: assertion 'self != NULL' failed

(com.github.lainsce.quilter:4801): Gtk-CRITICAL **: 17:33:24.725: gtk_header_bar_pack: assertion 'gtk_widget_get_parent (widget) == NULL' failed

** (com.github.lainsce.quilter:4801): CRITICAL **: 17:33:24.725: quilter_main_window_dynamic_margins: assertion 'self != NULL' failed

** (com.github.lainsce.quilter:4801): ERROR **: 17:33:24.743: SideBar.vala:194: Unable to read file: Failed to open file “/home/user/Documents/Short Story draft - Be.md”: No such file or directory
Trace/breakpoint trap
```

(I renamed the location path in the final line before pasting it into the GitHub window, for privacy's sake)

After moving the file back to the location Quilter remembered, the app now launches successfully.

To recreate the steps:
- Edit a file in Quilter so that it appears in the sidebar
- Close Quilter
- Move the file to a new folder in your file manager
- Reopen Quilter
- Attempt to close the file in the sidebar

**Comment by Hedronmx** [comment]
Can confirm, can't open quilter.
```
** (com.github.lainsce.quilter:3938): CRITICAL **: 19:34:52.350: gtk_spell_checker_set_language: assertion 'GTK_SPELL_IS_CHECKER (spell)' failed

** (com.github.lainsce.quilter:3938): CRITICAL **: 19:34:52.350: gtk_spell_checker_attach: assertion 'GTK_SPELL_IS_CHECKER (spell)' failed

** (com.github.lainsce.quilter:3938): CRITICAL **: 19:34:52.350: quilter_main_window_dynamic_margins: assertion 'self != NULL' failed

** (com.github.lainsce.quilter:3938): CRITICAL **: 19:34:52.376: quilter_main_window_dynamic_margins: assertion 'self != NULL' failed

** (com.github.lainsce.quilter:3938): CRITICAL **: 19:34:52.391: quilter_main_window_dynamic_margins: assertion 'self != NULL' failed

** (com.github.lainsce.quilter:3938): CRITICAL **: 19:34:52.406: quilter_main_window_dynamic_margins: assertion 'self != NULL' failed

** (com.github.lainsce.quilter:3938): CRITICAL **: 19:34:52.426: quilter_main_window_dynamic_margins: assertion 'self != NULL' failed

** (com.github.lainsce.quilter:3938): ERROR **: 19:34:52.445: SideBar.vala:194: Unable to read file: Failed to open file “/home/carlos/.htaccess”: No such file or directory
Trace/breakpoint trap
```

**Comment by lainsce** [comment]
I'll see what I can do for a new release

**Comment by lainsce** [comment]
Fixed in 1.8.1, soon released.
